### PR TITLE
sensors app: display available sensors

### PR DESCRIPTION
**Pull request by bradjc**
This:
- Prints the sensors which will be sampled.
- Updates the ambient light variable name.
- Fixes the print check for the temperature and humidity sensors.

### CODE REVIEW

**Review by hudson-ayers — APPROVED** [review]

File: examples/sensors/main.c [hunk 1]
```diff
@@ -28,7 +28,7 @@ static void timer_fired(__attribute__ ((unused)) int arg0,
                         __attribute__ ((unused)) int arg1,
                         __attribute__ ((unused)) int arg2,
                         __attribute__ ((unused)) void* ud) {
-  int light = 0;
+  int lite = 0;
```
Comment (alistair23) [inline examples/sensors/main.c:31]: Why `lite`?

Comment (alistair23) [reply examples/sensors/main.c:31]: `light_intensity` might be better

Comment (bradjc) [reply examples/sensors/main.c:31]: temp, humi, lite

**Pull request merged**